### PR TITLE
TASK: Adjust to `Node:$nodeType` removal

### PR DESCRIPTION
**Pull request by mhsdesign**
see https://github.com/neos/neos-development-collection/pull/5021

**What I did**

**How I did it**

**How to verify it**

### CODE REVIEW

File: Classes/Domain/Model/Changes/Property.php [hunk 1]
```diff
@@ -248,41 +245,38 @@ private function handleNodeReferenceChange(Node $subject, string $propertyName):
...
-    private function handleHiddenPropertyChange(Node $subject, string $propertyName): void
+    private function handleHiddenPropertyChange(Node $subject): void
     {
-        $value = $this->nodePropertyConversionService->convert(
-            $this->getNodeType($subject)->getPropertyType($propertyName),
-            $this->getValue()
-        );
+        // todo simplify conversion
+        /** @var bool $value */
+        $value = $this->nodePropertyConversionService->convert('boolean', $this->getValue());
```
Comment (mhsdesign) [inline Classes/Domain/Model/Changes/Property.php]:
```suggestion
        $value = (bool)$this->nodePropertyConversionService->convert('boolean', $this->getValue());
```

**Comment by mhsdesign** [comment]
Hurray ci passes again :D @grebaldi

**Review by grebaldi — APPROVED** [review]
Works like a charm, @mhsdesign :+1: 

It looks like we can merge this without the upstream change in https://github.com/neos/neos-development-collection/pull/5021, right?

**Comment by mhsdesign** [comment]
Yes that was the plan merge first to have it seamlessly work

**Pull request merged**